### PR TITLE
S8 Design Updates

### PR DESCRIPTION
**Pull request by DarkPurple141**
Still not done, merging now so don't lose track.

## Visual Summary
screen shot 2018-10-08 at 1 57 44 pm
screen shot 2018-10-08 at 1 57 49 pm
screen shot 2018-10-08 at 1 57 55 pm
screen shot 2018-10-08 at 1 58 15 pm

### CODE REVIEW

File: frontend/src/config.js [hunk 1]
```diff
@@ -6,7 +6,7 @@ import AppLogo from '@/components/AppLogo'
 import AppBreadCrumb from '@/components/AppBreadCrumb'
 
 Vue.config.productionTip = false
-Vue.config.devtools = true
```
[the remark this answers is not in the document]
Comment (DarkPurple141) [reply frontend/src/config.js]: Oh for the staging build. Yeah okkkkk.

File: frontend/src/views/course/index.vue [hunk 2]
```diff
@@ -1,27 +1,36 @@
...
+                <TabButton :active="this.$route.name == 'questions'">Questions</TabButton>
+            </router-link>
 
-        <router-link :to="{name: 'reviews'}">
-            <TabButton :active="this.$route.name == 'reviews'">Reviews</TabButton>
-        </router-link>
-    </div>
+        </div>
 
+        <div class="course-info">
```
Comment (NunoDasNeves) [inline frontend/src/views/course/index.vue]: I know it's not finished, but I just don't think this should be on the side. Plz put it back for now I think?

File: frontend/public/index.html [hunk 3]
```diff
@@ -1,18 +1,41 @@
...
+        'https://fonts.googleapis.com/css?family=Open+Sans', 
+        'https://fonts.googleapis.com/icon?family=Material+Icons'
+      ]
+      .map(href => {
+        const link = document.createElement('link')
+        link.href = href
+        link.setAttribute('rel', 'stylesheet')
+        return link
+      }).map(el => {
+        document.head.appendChild(el)
```
[the remark this answers is not in the document]
Comment (DarkPurple141) [reply frontend/public/index.html]: Links in the head are blocking by default. Google fonts seem to be affecting page load time, so better to defer, then add later, meanwhile app can get booted up and user probs won't even notice the difference.

File: frontend/src/router.js [hunk 4]
```diff
@@ -66,8 +58,8 @@ export default new Router({
           })
         },
         {
-          path: 'reviews',
-          name: 'reviews',
+          path: '/',
+          name: 'info',
```
[the remark this answers is not in the document]
Comment (DarkPurple141) [reply frontend/src/router.js]: It is now.

[the remark this answers is not in the document]
Comment (DarkPurple141) [reply frontend/src/router.js]: For now.

File: frontend/src/views/Design.vue [hunk 5]
```diff
@@ -49,7 +49,8 @@
                     :title="'This is a standard length title'"
                     :body="'This is a longer body, that includes a lot more words than the title, but isn\'t too long!'"
                 />
-                <AppRatingCircle/>
+                <AppRatingCircle :value="20"/>
+                <app-rating-bar :value="20"/>
```
[the remark this answers is not in the document]
Comment (DarkPurple141) [reply frontend/src/views/Design.vue]: Makes no difference but yeah.

File: frontend/src/views/course/index.vue (hunk 2 above)
Comment (DarkPurple141) [reply frontend/src/views/course/index.vue]: So remove entirely?

Comment (NunoDasNeves) [reply frontend/src/views/course/index.vue]: yeeah

**Comment by DarkPurple141** [comment]
Do u want to merge this? I just made some things to make it clear things are broken.

**Pull request merged**